app/models/util/file_manager.rb

```diff
@@ -171,5 +171,11 @@ def save_static_copy
       zip_file_name
     end
 
+    # 
+    def download_all_studies(unzip_location)
+      system(`wget -c -q --show-progress -P "${unzip_location}" https://clinicaltrials.gov/AllAPIJSON.zip`)
```

Comment:
add some error checking here so that an exception is thrown if we don't get a successful download

```ruby
success = system(`wget -c -q --show-progress -P "${unzip_location}" https://clinicaltrials.gov/AllAPIJSON.zip`)
raise "Could not download file" unless success
```